recipes/statannot/LICENSE.txt

```diff
@@ -0,0 +1,22 @@
+MIT License
```

Comment:
It looks like `LICENSE` is already included in the PyPI tarball, so you can remove this.